package.json

```diff
@@ -68,7 +68,7 @@
     "lint:prettier": "prettier --list-different .",
     "reload:server": "pm2 reload api-server/ecosystem.config.js",
     "preseed": "npm-run-all create:shared",
-    "playwright:install-build-tools": "cd ./e2e && npm i && npx playwright install && npx playwright install-deps",
+    "playwright:install-build-tools": "npx playwright install --with-deps",
     "playwright:install-build-tools-linux": "sh ./playwright-install.sh",
```

Comment:
It's a little tangential, but do you know if this is necessary? I don't recall needing it.

Reply:
Okay, cool. I'll put a quick PR together to remove that :)